modules/math/src/main/java/com/opengamma/strata/math/impl/statistics/descriptive/ExponentiallyWeightedInterpolationQuantileMethod.java

```diff
@@ -130,18 +126,19 @@ private QuantileResult quantileDetails(
 
   /**
    * Computes value-at-risk.
-   * @param index  the index from which the VaR should be computed
-   * @param runningWeight  the running weight up to index
-   * @param isExtrapolated  flag indicating if value should be extrapolated (flat) beyond the last value
-   * @param s  the sorted sample
-   * @param w  the sorted weights
-   * @param order  the order of the sorted sample in the unsorted sample
-   * @param level  the level at which the VaR should be computed
+   *
+   * @param index the index from which the VaR should be computed
```

Comment:
Two spaces between parameter name and comment.
The same comments for the other parameters.